docs/init-repo.md

```diff
@@ -0,0 +1,38 @@
+# init-repo.ps1
+Repository initialization.
+
+Input parameters:
+* **$AzureDevOpsCollection**: url to Azure DevOps, e.g. https://dev.azure.com/oriflame/ (optional)
+* **$DevOpsProject**: name of the project in Azure DevOps, e.g. http://dev.azure.com/oriflame/{THIS}
+* **$RepositoryName**: name of the project in Azure DevOps, e.g. http://dev.azure.com/oriflame/something/_git/{THIS}
+
+It does following:
+
+* check & install prerequisities: 
+  * in case GIT command is missing it installs [chocolatey](https://chocolatey.org/) and [git](https://chocolatey.org/packages/git.install)
+  * in case TF tool is missing (C:\Program Files (x86)\Microsoft Visual Studio\2019\TeamExplorer\Common7\IDE\CommonExtensions\Microsoft\TeamFoundation\Team Explorer\tf.exe is searched) it installst [Team Explorer](https://chocolatey.org/packages/visualstudio2017teamexplorer)
+* it checks, if we are in GIT repo and if not, it clones the repo locally from provided project and repository name
+* it switches to **master** branch
+* if **develop** branch is not yet created it is
```

Comment:
it is what? ;) created?

Reply:
it **will** be created..in the future.. This is not yet done, so it will throw an error like "Please create xxx-CI build as it was not found." However nice catch, I have to fix the MD. thanks!

Reply:
fixed